Resources/Textures/Mobs/Customization/Moth/moth_wings.rsi/meta.json

```diff
@@ -130,6 +130,14 @@
     {
       "name": "underwing_secondary",
       "directions": 4
+    },
+    {
+       "name": "membral_primary",
+       "directions": 4
+    },
+    {
+        "name": "membral_secondary",
+        "directions": 4
```

Comment:
Instead of modifying upstream's RSI, you should create a new RSI in the _CD directory.